gstreamer/src/tags.rs

```diff
@@ -594,6 +595,8 @@ mod serde {
                 type_ => {
                     if DateTime::static_type() == type_ {
                         ser_tag!(serializer, self, DateTime)
+                    } else if Sample::static_type() == type_ {
+                        ser_tag!(serializer, self, DateTime)
```

Comment:
This should be `Sample` :)

Reply:
Oups! :) I didn't implement it yet anyway. Will be available soon.